src/dotnet/CarbonAware.LocationSources.Azure/test/AzureLocationSourceTest.cs

```diff
@@ -0,0 +1,97 @@
+using CarbonAware.Model;
+using NUnit.Framework;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Moq.Protected;
+using CarbonAware.Exceptions;
+
+namespace CarbonAware.LocationSources.Azure.Test;
+
+public class AzureLocationSourceTest
+{   
+    [Test]
+    public async Task TestToGeopositionLocation()
```

Comment:
NIT: prefer full method name in test and the condition under test like `TestToGeopositionLocationAsync_ValidLocation`

Reply:
Out of pure curiosity - what was the the resolution here?